src/integrationTest/sam.test.ts

```diff
@@ -103,70 +105,57 @@ async function closeAllEditors(): Promise<void> {
 async function activateExtensions(): Promise<void> {
     console.log('Activating extensions...')
     await activateExtension(VSCODE_EXTENSION_ID.python)
-    await activateExtension(VSCODE_EXTENSION_ID.awstoolkit)
     console.log('Extensions activated')
 }
 
 async function configurePythonExtension(): Promise<void> {
-    logSeparator()
     const configPy = vscode.workspace.getConfiguration('python')
     // Disable linting to silence some of the Python extension's log spam
     await configPy.update('linting.pylintEnabled', false, false)
     await configPy.update('linting.enabled', false, false)
-    logSeparator()
 }
 
 async function configureAwsToolkitExtension(): Promise<void> {
-    logSeparator()
     const configAws = vscode.workspace.getConfiguration('aws')
     await configAws.update('logLevel', 'verbose', false)
```

Comment:
The change to set the max log level may mean this `loglevel` configuration change is no longer necessary